schema/C2_LOAN_GUARANTOR.json

```diff
@@ -1,412 +1,459 @@
-            "fec_spec": {
-                "COL_SEQ": 2,
-                "FIELD_DESCRIPTION": "FILER COMMITTEE ID NUMBER",
-                "TYPE": "A/N-9",
-                "REQUIRED": "X (error)",
-                "SAMPLE_DATA": "C00123456"
-            }
-        },
-        "transaction_type_identifier": {
-            "title": "TRANSACTION TYPE IDENTIFIER",
-            "description": "",
-            "const": "C2_LOAN_GUARANTOR",
-            "examples": [
-                "C2_LOAN_GUARANTOR"
-            ],
-            "fec_spec": {
-                "FIELD_DESCRIPTION": "TRANSACTION TYPE IDENTIFIER",
-                "TYPE": "A/N-12",
-                "REQUIRED": "X (error)",
-                "SAMPLE_DATA": "C2_LOAN_GUARANTOR"
-            }
-        },
-        "transaction_id": {
-            "title": "TRANSACTION ID NUMBER",
-            "description": "",
-            "type": "string",
-            "minLength": 1,
-            "maxLength": 20,
-            "pattern": "^[ -~]{0,20}$",
-            "examples": [
-                "C123456789-3456-001"
-            ],
-            "fec_spec": {
-                "COL_SEQ": 3,
-                "FIELD_DESCRIPTION": "TRANSACTION ID NUMBER",
-                "TYPE": "A/N-20",
-                "REQUIRED": "X (error)",
-                "SAMPLE_DATA": "C123456789-3456-001",
-                "RULE_REFERENCE": "must be unique and UPPER CASE for the life of the report (original + all amendments)",
-                "NOTE": "May be a combination of Parent SC/ TranID + a unique ID for this Child SC2/ record"
-            }
-        },
-        "back_reference_tran_id_number": {
-            "title": "BACK REFERENCE TRAN ID NUMBER",
-            "description": "",
-            "type": "string",
-            "minLength": 1,
-            "maxLength": 20,
-            "pattern": "^[ -~]{0,20}$",
-            "examples": [
-                "C123456789-3456"
-            ],
-            "fec_spec": {
-                "COL_SEQ": 4,
-                "FIELD_DESCRIPTION": "BACK REFERENCE TRAN ID NUMBER",
-                "TYPE": "A/N-20",
-                "REQUIRED": "X (error)",
-                "SAMPLE_DATA": "C123456789-3456",
-                "RULE_REFERENCE": "Reference to the Tran ID of a Related SC/{***} Record",
-                "NOTE": "References the associated SC transaction Loan Received"
-            }
+  "$schema": "https://json-schema.org/draft-07/schema#",
+  "$id": "https://github.com/fecgov/fecfile-validate/blob/main/schema/C2_LOAN_GUARANTOR.json",
+  "version": "8.3.0.1",
+  "title": "C2 Loan Guarantor",
+  "description": "SCHEDULE C2 - LOAN GUARANTOR NAME & ADDRESS INFORMATION (SUPPLEMENTARY FOR INFORMATION FOUND ON SCHEDULE C)",
+  "type": "object",
+  "required": [
+    "form_type",
+    "filer_committee_id_number",
+    "transaction_type_identifier",
+    "transaction_id",
+    "back_reference_tran_id_number",
+    "entity_type",
+    "guarantor_last_name",
+    "guarantor_first_name",
```

Comment:
`guarantor_first_name` and `guarantor_last_name` should be removed from the required list